src/main/java/me/zeroeightsix/kami/module/modules/movement/AutoWalk.kt

```diff
@@ -41,28 +49,30 @@ class AutoWalk : Module() {
                 disableBaritone = false
                 event.movementInput.moveForward = -1f
             }
-            AutoWalkMode.BARITONE -> disableBaritone = true
+            AutoWalkMode.PATHFIND -> {
+                disableBaritone = true
+            }
         }
     })
 
     @EventHandler
     private val kickListener = Listener(EventHook { event: ServerDisconnectedEvent? ->
-        if (mode.value == AutoWalkMode.BARITONE && isEnabled) {
+        if (mode.value == AutoWalkMode.PATHFIND && isEnabled) {
             disable()
         }
     })
 
     public override fun onEnable() {
-        if (mode.value != AutoWalkMode.BARITONE) return
+        if (mode.value != AutoWalkMode.PATHFIND) return
 
         if (mc.player == null) {
             disable()
             return
         }
 
-        if (mc.player.isElytraFlying) {
-            sendErrorMessage("$chatName Baritone mode isn't currently compatible with Elytra flying! Choose a different mode if you want to use AutoWalk while Elytra flying")
-            disable()
+        if (MODULE_MANAGER.isModuleEnabled(ElytraFlight::class.java)) {
```

Comment:
There should probably be an option called "ElytraFlightCheck", so you don't need to use KAMI Blue's elytraflight. It is the best but I like stuff being as customizable as possible.